packages/gatsby-theme-docs/README.md

```diff
@@ -82,6 +82,8 @@ The project structure should contain at least the following files and folders:
 
   - `overrideDefaultConfigurationData` (_optional_, array of glob strings): allows to replace the configuration files in `src/data` instead of augmenting them. The option is passed to the `ignore` [option of the gatsby filesystem plugin](https://www.gatsbyjs.org/packages/gatsby-source-filesystem/#options). For example, by passing `['**/top-*']` and placing `top-menu.yaml` and `top-side-menu.yaml` files in the website's `src/data` folder the top navigation can be overridden completely. If this option is used, the files matching the glob patterns **must** be provided.
 
+  - `themeAddOns` (_optional_, array of Gatsby Theme plugin names): see [using Theme with Add-Ons](#using-theme-with-addons).
```

Comment:
since it's in the context of the "theme-docs" plugin, I think the name of this can be only `addOns`.